components/identity-mgt/org.wso2.carbon.identity.mgt/src/test/java/org/wso2/carbon/identity/mgt/listener/TenantSyncListenerTest.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.mgt.listener;
+
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.testng.annotations.BeforeMethod;
+import org.wso2.carbon.identity.mgt.constants.IdentityMgtConstants.TenantManagement;
+import org.wso2.carbon.identity.mgt.dto.TenantManagementEventDTO;
+import org.wso2.carbon.stratos.common.beans.TenantInfoBean;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
We are using testng framework. Shall we using assert functions of testng

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/7075/commits/33ae41f3ec39647902d1b4cd5bdb0d29fd058d70